### PR TITLE
ENDOC-594 misc 7.1 updates

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

**Review by nshaw — APPROVED** [review]

File: vuepress/docs/next/docs/getting-started/ent-bundle.md [hunk 1]
```diff
@@ -167,8 +167,9 @@ The following ent commands are used to manage git-based (< Entando 7.1) bundles.
 |`ent prj install`| Install the bundle into Entando |
 |`ent prj install --conflict-strategy=OVERRIDE`| Adopt a strategy for conflicts affecting installed bundles |
 |`ent prj pbs-init` | Initialize the bundle directory |
-|`ent prj pbs-publish`| Publish the artifacts to GitHub and Docker Hub |
+|`ent prj pbs-publish`| Publish the artifacts to GitHub and Docker Hub |  
 
+**Note**: ent prj commands are deprecated in favor of ent bundle commands.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/getting-started/ent-bundle.md:172]: backtick `ent prj` and `ent bundle` (commands are backticked); no backticking required in "Entando project commands" or "project commands" but whenever a command or part of a command (like an option or argument) is written in its executable form it should be backticked

File: vuepress/docs/next/docs/curate/troubleshooting-guide.md [hunk 2]
```diff
@@ -14,45 +14,26 @@ kubectl logs -f deployment/quickstart-server-deployment --namespace YOUR-NAMESPA
...
 **Bundle installation fails due to plugin images that are not reachable**
 
 A bundle installation does not complete successfully because a plugin in a bundle, defined by a Docker image, is not available. 
 
 ### Solution
-This may happen if the Docker image for the plugin is located in a private registry or not yet published. Verify that the Docker image you are referencing is published, correctly formatted, and publicly available.
-
-Then 
-[publish a new bundle version](#how-do-i-publish-a-new-version) as explained above.
+This may happen if the Docker image for the plugin is located in a private registry or not yet published. Verify that the Docker image you are referencing is published, correctly formatted, and publicly available. Then, publish the updated bundle with the ent bundle publish command.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/curate/troubleshooting-guide.md:36]: "...with the command `ent bundle publish`" or "Then, use `ent bundle publish` to publish the updated bundle."

File: vuepress/docs/next/docs/curate/troubleshooting-guide.md [hunk 3]
```diff
@@ -14,45 +14,26 @@ kubectl logs -f deployment/quickstart-server-deployment --namespace YOUR-NAMESPA
...
 ERROR - File with name {filename} not found in the bundle
 ```
 
 ### Solution
 
-Verify that the component named in the descriptor file is actually present in the bundle, in the location specified, and that the reference is properly formatted.
-
-Then 
-[publish a new bundle version](#how-do-i-publish-a-new-version) as explained above.
+Verify that the component named in the descriptor file is actually present in the bundle, in the location specified, and that the reference is properly formatted. Then, publish the updated bundle with the ent bundle publish command.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/curate/troubleshooting-guide.md:28]: can trim: "Verify that the component named in the descriptor file is present in the bundle at the specified location and that the reference is properly formatted." or "Verify that the component named in the descriptor file is at the specified bundle location and that the reference is properly formatted."

"...with the command ent bundle publish" or "Then, use ent bundle publish to publish the updated bundle."

File: vuepress/docs/next/docs/curate/troubleshooting-guide.md [hunk 4]
```diff
@@ -14,45 +14,26 @@ kubectl logs -f deployment/quickstart-server-deployment --namespace YOUR-NAMESPA
...
-2. Once the new tag is published, update the bundle custom resource in the cluster by adding the new tag to the `tags` object and replacing the latest `dist-tags` to point to the new version.
-3. Then proceed with the new installation.
-
-Note: If you're actively working on your bundle and simply want to verify that things are working correctly, you can override a specific tag using the Git command `git tag -f` instead of generating a new tag for each release of the bundle. You should follow this practice only during development, never in production.
-
 ## ERROR - File not found in bundle
 **Installation fails because a file has not been found in the bundle**
 
-When a component that is referenced in the descriptor.yaml is missing in the bundle or not properly called, the bundle installation will fail and the error reported in the logs.
+When a component that is referenced in the entando.json is missing in the bundle or not properly called, the bundle installation will fail and the error reported in the logs.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/curate/troubleshooting-guide.md:20]: tenses don't match: "installation fails and the error is reported.."

File: vuepress/docs/next/docs/curate/bundle-presentation-config.md [hunk 5]
```diff
@@ -27,17 +27,15 @@ spec:
...
-    thumbnail: >-
-      data:image/jpeg;base64,....
   tags:
-    # ....
+   - tarball: docker://registry.hub.docker.com/entando/simple-entando-bundle
+     version: 0.0.1
 ```
 
-To change the UI elements, update the custom resource in accordance with this table:
+Spec Definitions:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/curate/bundle-presentation-config.md:35]: why is the sentence this replaces no longer relevant?

**Pull request merged**